src/utils/container.ts

```diff
@@ -18,10 +18,10 @@ export type ContainerGetter<TContext extends object> = (
 class DefaultContainer {
   private instances: Array<{ type: Function; object: any }> = [];
 
-  get<T>(someClass: SupportedType<T>): T {
+  get<T>(someClass: SupportedType<T>, resolverData: ResolverData): T {
     let instance = this.instances.find(it => it.type === someClass);
     if (!instance) {
-      instance = { type: someClass, object: new (someClass as any)() };
+      instance = { type: someClass, object: new (someClass as any)(resolverData) };
```

Comment:
I can't accept implicit calling the constructor with resolver data as an argument.

If you need access to resolver data in container, just use the scoped containers feature:
https://typegraphql.ml/docs/dependency-injection.html#scoped-containers

The default container is as simple as fuck - just to make base things works without 3rd party container. If you need more - provide your own container.

Reply:
thx~ yep I can use my own container.